django/core/secret_key.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
```

Comment:
`django.core.secret_keys` ?

Reply:
Not relevant anymore since this PR no longer contains the settings backend stuff. :)